net/suricata6/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2006-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Omit or assign it to your own.
More details: https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L45